src/behaviors/Pressable.tsx

```diff
@@ -9,16 +13,22 @@ export type PressableProps = PublicBoxProps & PressableBoxProps;
 // the Pressable component an instance of a Box component. This may or may not be the right approach
 // to take, but it seemed more composable to do this than to create a component that attempted to
 // manipulate its child components or added an additional DOM layer
-export const Pressable = ({children,
-                           onPress, 
-                           className = '', 
-                           spacingSize = 'MEDIUM',
-                           spacingStyle = 'INSET'}: PressableProps) => (
-  <AbstractBox boxType="PRESSABLE"
-               className={`pressable ${className}`}
-               onPress={onPress}
-               spacingSize={spacingSize}
-               spacingStyle={spacingStyle}>
+export const Pressable = ({
+  children,
+  onPress,
+  className = "",
+  spacingSize = "MEDIUM",
+  spacingStyle = "INSET",
+  spacingAlignment = "LEFT",
+}: PressableProps) => (
+  <AbstractBox
```

Comment:
An interesting (performance wise) approach to "make some abstract thing less abstract" is to threat component here as a function (it is a function)
```tsx
export const Pressable = ({
  // stuff
}: PressableProps) => (
  AbstractBox({
    boxType:"PRESSABLE"
    className:{`pressable ${className}`}
    onPress,
    spacingSize,
    spacingStyle,
    spacingAlignment:spacingAlignment,
    children
  >
})
)
```
One `createElement` less, one transparent `fiber` less. One crappy node in DevTools less. Strongly typed for forwardRef.

Reply:
That's really interesting and a great tip! Thanks for sharing that suggestion! :)